steps/templating.md

```diff
@@ -3,71 +3,12 @@ layout: default
 ---
 # Templates
 
-To combine data and html we use a template engine.
+To make creating views in Express one can use various different template engines; to makes it easy to combine data into html files. We will use Handlebars as our template engine.
```

Comment:
To create views in Express you one can use various different template engines; this makes it easy to combine data into HTML files. We will use Handlebars as our template engine.